### ISSUE TITLE
0.13 breaks arrays

### ISSUE THREAD

**Issue by sebhoss** [dadav/helm-schema#69]
I have the following in my values.yaml:

```yaml
  # @schema
  # required: false
  # @schema
  args:
    - start
```

helm-schema version prior to 0.13 produced this schema which works fine with helm itself:

```json
"args": {
  "items": {
    "anyOf": [
      {
        "type": "string"
      }
    ]
  },
  "title": "args"
}
```

helm-schema 0.13 produces this:

```json
"args": {
  "items": {
    "anyOf": [
      {
        "required": [],
        "type": "string"
      }
    ],
    "required": [],
    "type": "array"
  },
  "required": [],
  "title": "args"
}
```

The produced schema does not accept the original values.yaml file since it expects the values of `args` to be of type `array` but they are of type string. I can fix this myself by specifying the type of the items like this:

```yaml
  # @schema
  # required: false
  # type: array
  # items:
  #   type: string
  # @schema
  args:
    - start
```

**Comment by dadav** [comment]
Should be fixed in `0.13.1-2`

**Comment by sebhoss** [comment]
yep that works fine - thanks a lot for the quick response!
